Google.Cloud.EntityFrameworkCore.Spanner/Query/Internal/SpannerQuerySqlGenerator.cs

```diff
@@ -122,5 +137,54 @@ protected override Expression VisitSqlFunction(SqlFunctionExpression sqlFunction
 
             return base.VisitSqlFunction(sqlFunctionExpression);
         }
+
+        protected virtual Expression VisitContains(SpannerContainsExpression containsExpression)
```

Comment:
The logic here seems to overlap to a large degree with the logic in `SpannerContainsExpression`. Could we combine the equal parts into one method?

Reply:
I agree--I found it strange that EF Core required two separate methods that essentially do the same thing (building a SQL string). However, one uses an `ExpressionPrinter`, while the other uses an `IRelationalCommandBuilder`. Looking at the built-in commands in EFCore, as well as the implementations in the postgres provider, it seems that separating the logic is common practice.

Reply:
Ack